src/System.Private.CoreLib/src/System/RtType.cs

```diff
@@ -4440,84 +4455,45 @@ private void CreateInstanceCheckThis()
         }
 
         // the cache entry
-        private class ActivatorCacheEntry
+        private sealed class ActivatorCacheEntry
```

Comment:
Nit: maybe we can drop `Entry` from the name since this is no longer an entry in an array?